cdc/kv/region_worker.go

```diff
@@ -541,7 +558,7 @@ func (w *regionWorker) eventHandler(ctx context.Context) error {
 			// We measure whether the current worker is busy based on the input
 			// channel size. If the buffered event count is larger than the high
 			// watermark, we send events to worker pool to increase processing
-			// throughput. Otherwise we process event in local region worker to
+			// throughput. Otherwise, we process event in local region worker to
```

Comment:
will  the value of the  highWatermarkMet be affected ?

Reply:
yes , the batch will affect the highWatermarkMet, but i think it's ok to use old highWatermarkMet , because  data event is not batch for now